### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xitonix**

### CODE REVIEW

**Comment by sandbox-alg** [comment]
sdsdsd

File: README.md [hunk 1]
```diff
@@ -26,4 +26,3 @@ Once you installed the tool, you can simply use `ssh` or `scp` the normal way.
 ssh -i "identity.pem" user@host
```
Comment (sandbox-alg) [inline README.md:26]: asdasd

Comment (sandbox-alg) [reply README.md:26]: sdfsdf

Comment (sandbox-alg) [reply README.md:26]: sdfsdf

Comment (sandbox-alg) [reply README.md:26]: fdgdfg

Comment (sandbox-alg) [reply README.md:26]: sdfsdf

Comment (sandbox-alg) [reply README.md:26]: dfgdfg

Comment (sandbox-alg) [reply README.md:26]: asdasd

**Comment by sandbox-alg** [comment]
sdfsdf

File: README.md (hunk 1 above)
Comment (sandbox-alg) [reply README.md:26]: sdfsdfsdf

Comment (sandbox-alg) [reply README.md:26]: sdfsdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]

**Review by sandbox-alg — APPROVED** [review]
sdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
werwer

**Review by sandbox-alg — COMMENTED** [review]
ertert

**Review by sandbox-alg — COMMENTED** [review]
asdasd

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdf

**Review by sandbox-alg — APPROVED** [review]
sdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
asdasd

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdf

**Review by sandbox-alg — APPROVED** [review]

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdfs

**Review by sandbox-alg — APPROVED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
werwer

**Review by sandbox-alg — APPROVED** [review]
asdasd

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdfs

**Review by xitonix — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — COMMENTED** [review]
sdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdfs

**Pull request merged**